.travis.yml

```diff
@@ -67,6 +67,11 @@ install:
   - export CGO_LDFLAGS="$CGO_LDFLAGS -L`pwd` -lrocksdb"
   - export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:`pwd`"
 
+before_script:
+  - export CHERAMI_PACKAGES=`go list ./... | grep -ve vendor | grep -ve test | sed "s#github.com/uber/cherami-server#.#"` 
```

Comment:
consider moving these to a target in Makefile instead, that way, one can run this locally as well. In addition, you can reuse the #defines that already compute the set of files/packages to compile